CMakeLists.txt

```diff
@@ -142,8 +145,22 @@ include(CmakeFunctionHelper)
 if (INSTALL_HEADER_ONLY)
   add_library(portblas INTERFACE)
   set_target_properties(portblas PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES "${PORTBLAS_INCLUDE};$<INSTALL_INTERFACE:src>"
-  )
+    INTERFACE_INCLUDE_DIRECTORIES "${PORTBLAS_INCLUDE};${PORTBLAS_INSTALL_SRC}")
+  include(CheckCXXCompilerFlag)
+  check_cxx_compiler_flag("-fsycl" is_dpcpp)
+  if(is_dpcpp)
+    target_compile_definitions(portblas INTERFACE "SB_ENABLE_USM")
+  endif()
+  if(${BLAS_ENABLE_COMPLEX})
+    target_compile_definitions(portblas INTERFACE "BLAS_ENABLE_COMPLEX")
+  endif()
+  target_compile_definitions(portblas INTERFACE ${TUNING_TARGET})
+  if((${CMAKE_CXX_COMPILER_ID} STREQUAL "IntelLLVM") AND NOT
+    (${TUNING_TARGET} STREQUAL "INTEL_GPU") )
+      target_compile_options(portblas INTERFACE -fno-fast-math)
+      target_compile_options(portblas INTERFACE -mllvm -loopopt=0 )
```

Comment:
Maybe, it is better use `-mllvm=-loopopt=0` in order to avoid possible issues due to CMake option deduplication. 

*The same apply to `cmake/Modules/FindDPCPP.cmake`.*